.github/CODEOWNERS

```
@@ -2,4 +2,13 @@
 * @alphagov/di-dev-platform @alphagov/digital-identity-performance-test
 
 # Test Scripts
-/deploy/scripts @alphagov/digital-identity-performance-test
\ No newline at end of file
+/deploy/scripts @alphagov/digital-identity-performance-test
+
+# Team Specific Test Script Directories
+/deploy/scripts/src/accounts @alphagov/digital-identity-performance-test @alphagov/gov-uk-accounts
+/deploy/scripts/src/authentication @alphagov/digital-identity-performance-test @alphagov/digital-identity-authentication
```

```suggestion
/deploy/scripts/src/authentication @alphagov/digital-identity-performance-test
```
Remove if team is inaccessible